pytx3/pytx3/collab_config.py

```diff
@@ -42,10 +39,9 @@ def default_state_dir_name(self) -> str:
     def load(cls, file: t.IO):
         content = json.load(file)
         return cls(
-            content[cls.KEY_NAME],
+            content.get(cls.KEY_NAME, "Sample Config"),
             content[cls.KEY_LABELS],
```

Comment:
Since labels is required in the config and not the other fields, do we want to raise an exception here with a descriptive message that the config must include the labels information? This is what the stack trace looks like right now.

```
$ pytx3 fetch 
Traceback (most recent call last):
  File "<clip>/bin/pytx3", line 11, in <module>
    load_entry_point('pytx3', 'console_scripts', 'pytx3')()
  File "<clip>/pytx3/pytx3/cli/main.py", line 137, in main
    execute_command(namespace)
  File "<clip>/pytx3/pytx3/cli/main.py", line 69, in execute_command
    cfg = init_config_file(namespace.config)
  File "<clip>/pytx3/pytx3/cli/main.py", line 124, in init_config_file
    return CollaborationConfig.load(f)
  File "<clip>/pytx3/pytx3/collab_config.py", line 43, in load
    content[cls.KEY_LABELS],
KeyError: 'labels'
```

Reply:
Will punt for a future change, I haven't figured out what the flow for creating/editing configs should be.